server/converters/diffexp_precompute.py

```diff
@@ -0,0 +1,144 @@
+"""This program precomputes data to speed up diffexp."""
+
+import sys
+import os
+import tiledb
+import argparse
+import numpy as np
+import shutil
+from server.common.app_config import AppConfig
+from server.data_common.matrix_loader import MatrixDataLoader
+from server.compute.diffexp_generic import compute_indices_hash, compute_indices_hash_version
+from server.compute.diffexp_cxg import compute_mean_var
+from server.data_cxg.cxg_adaptor import CxgAdaptor
+from server.common.errors import DatasetAccessError, ComputeError
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("dataset", help="input CXG dataset")
+    parser.add_argument("--max_groups", "--max", type=int, default=100)
+    parser.add_argument("--min_groups", "--min", type=int, default=2)
+    parser.add_argument("--max_rows", "--maxr", type=int, default=1000000)
```

Comment:
why have a max?  Seems like the bigger the row set, the more we need the precompute?